go/vt/tabletmanager/gorpctmserver/gorpc_server.go

```diff
@@ -35,6 +36,7 @@ type TabletManager struct {
 
 // Ping wraps RPCAgent.
 func (tm *TabletManager) Ping(ctx context.Context, args, reply *string) error {
+	ctx = callinfo.RPCWrapCallInfo(ctx)
```

Comment:
Why is this needed only for TabletManager and not for others like SqlQuery, VTGate, etc?